services-camel-app/src/test/java/edu/unc/lib/boxc/services/camel/images/ImageEnhancementsRouterTest.java

```diff
@@ -90,64 +92,74 @@ public void cleanup() throws IOException {
     public void testThumbnailMulticast() throws Exception {
         createContext(thumbnailRoute);
 
-        getMockEndpoint("mock:direct:small.thumbnail").expectedMessageCount(1);
-        getMockEndpoint("mock:direct:large.thumbnail").expectedMessageCount(1);
+        var smallEndpoint = getMockEndpoint("mock:direct:small.thumbnail");
+        smallEndpoint.expectedMessageCount(1);
+        var largeEndpoint = getMockEndpoint("mock:direct:large.thumbnail");
+        largeEndpoint.expectedMessageCount(1);
         template.sendBodyAndHeaders("", createEvent(fileID, eventTypes, "false"));
 
-        assertMockEndpointsSatisfied();
+        smallEndpoint.assertIsSatisfied();
+        largeEndpoint.assertIsSatisfied();
     }
 
     @Test
     public void testThumbMulticastFilter() throws Exception {
         createContext(thumbnailRoute);
 
-        getMockEndpoint("mock:direct:small.thumbnail").expectedMessageCount(0);
-        getMockEndpoint("mock:direct:large.thumbnail").expectedMessageCount(0);
+        var smallEndpoint = getMockEndpoint("mock:direct:small.thumbnail");
+        smallEndpoint.expectedMessageCount(0);
+        var largeEndpoint = getMockEndpoint("mock:direct:large.thumbnail");
+        largeEndpoint.expectedMessageCount(0);
 
         Map<String, Object> headers = createEvent(fileID, eventTypes, "false");
         headers.put(CdrBinaryMimeType, "plain/text");
 
         template.sendBodyAndHeaders("", headers);
 
-        assertMockEndpointsSatisfied();
+        smallEndpoint.assertIsSatisfied();
+        largeEndpoint.assertIsSatisfied();
     }
 
     @Test
     public void testThumbDisallowedImageType() throws Exception {
         createContext(thumbnailRoute);
 
-        getMockEndpoint("mock:direct:small.thumbnail").expectedMessageCount(0);
-        getMockEndpoint("mock:direct:large.thumbnail").expectedMessageCount(0);
+        var smallEndpoint = getMockEndpoint("mock:direct:small.thumbnail");
+        smallEndpoint.expectedMessageCount(0);
+        var largeEndpoint = getMockEndpoint("mock:direct:large.thumbnail");
+        largeEndpoint.expectedMessageCount(0);
 
         Map<String, Object> headers = createEvent(fileID, eventTypes, "false");
         headers.put(CdrBinaryMimeType, "image/vnd.fpx");
 
         template.sendBodyAndHeaders("", headers);
 
-        assertMockEndpointsSatisfied();
+        smallEndpoint.assertIsSatisfied();
+        largeEndpoint.assertIsSatisfied();
     }
 
     @Test
     public void testThumbSmallRouteNoForceNoFileExists() throws Exception {
         when(addSmallThumbnailProcessor.needsRun(any())).thenReturn(true);
         createContext(smallThumbRoute);
 
-        getMockEndpoint("mock:exec:/bin/sh").expectedMessageCount(1);
+        var shEndpoint = getMockEndpoint("mock:exec:/bin/sh");
```

Comment:
it is unclear what this variable is short for? what's a shendpoint?

Reply:
it's short for "/bin/sh" since it is mocking that and I think there were already a couple methods in this class that called it shEndpoint, so I opted to use the same name everywhere